packages/shared/lib/core/network/utils/buildChainFromNetwork.ts

```diff
@@ -0,0 +1,13 @@
+import { selectedAccount } from '@core/account/stores'
+import { activeProfile } from '@core/profile'
+import { get } from 'svelte/store'
+import { networkStatus } from '../stores'
+import { ConnectedChain } from '../interfaces'
+
+export function buildChainFromNetwork(): ConnectedChain {
```

Comment:
Hmm we have two options here:
1. Keep as a util but pass parameters so that it doesn't read from a store
2. Turn into an action function 

I think for now 2 is easiest, although this has a "util" feel to me if that makes sense. 1 would mean we pass the stores in from the component, which may be nice for reactivity (e.g. if we switch an account), but I guess to access the account switcher the side drawer has to close anyway... :thinking:

Reply:
this is only an mvp so that we can create the list of chains. In another step (maybe once we can actually add iscp chains) this has to be generalized such that we can build a persisted store.

for now i will go for 1, because like this its a bit of a blackbox, which i think is sometimes harder to understand